lib/msf/core/post/unix.rb

```diff
@@ -82,8 +85,33 @@ def enum_user_directories
     user_dirs.compact!
     user_dirs.sort!
     user_dirs.uniq!
-
     user_dirs
   end
 
+  #
+  # It returns the username of the current user
+  # @return [String] with username
+  #
+  def whoami()
```

Comment:
Style. No need to use brackets `()` in method definitions when the method takes no arguments.